src/components/ReportingConfig/EmailDeliveryMethodForm.jsx

```diff
@@ -22,13 +22,12 @@ const EmailDeliveryMethodForm = ({ invalidFields, config, handleBlur }) => {
             id="email"
             name="emailRaw"
             defaultValue={config ? config.email.join('\n') : undefined}
-            onBlur={e => handleBlur(e, () =>
-              !isEmpty(e.target.value.split('\n')
-                .filter(email => !isEmail(email))))
-            }
+            onBlur={e => handleBlur(e, () => !isEmpty(e.target.value.split('\n')
```

Comment:
nit: something like this might be a bit more readable?

```jsx
onBlur={e => handleBlur(e, () => {
  const rows = e.target.value.split('\n');
  const emails = rows.filter(email => !isEmail(email));
  return !isEmpty(emails)
})}
```